Dockerfile

```diff
@@ -1,71 +1,54 @@
-# Build compilation image
-FROM ruby:3.1.4-alpine as builder
+FROM ruby:3.1.4-alpine3.18 as base
 
-# The application runs from /app
 WORKDIR /app
 
-# Add the timezone as it's not configured by default in Alpine
+# Add the timezone as it’s not configured by default in Alpine
 RUN apk add --update --no-cache tzdata && \
   cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
   echo "Europe/London" > /etc/timezone
 
-# build-base: complication tools for bundle
+# build-base: compilation tools for bundle
 # yarn: node package manager
 # postgresql-dev: postgres driver and libraries
-RUN apk add --no-cache build-base yarn postgresql-dev git
+RUN apk add --no-cache build-base yarn postgresql-dev git bash
 
-# Install bundler to run bundle exec
-# This should be the same version as the Gemfile.lock
+# Bundler version should be the same version as what the Gemfile.lock was bundled with
 RUN gem install bundler:2.3.14 --no-document
 
-# Install gems defined in Gemfile
 COPY .ruby-version Gemfile Gemfile.lock /app/
+RUN bundle config set without "development"
```

Comment:
We're installing all default and test group gems as standard

Presumably we might want the test group gems in case we want to run application tests on the containers built for any env, but wanted to check if you agree?

Reply:
I don't think we want test group gems on our production instance - they can be a vector for security issues, and also add a bit of bloat (although to be fair not masses of it).